src/components/RoomSelector.vue

```diff
@@ -203,98 +220,44 @@ export default {
 </script>
 
 <style lang="scss" scoped>
-
-:deep(.modal-container) {
-	height: 700px;
-}
-
-/* FIXME: remove after https://github.com/nextcloud-libraries/nextcloud-vue/pull/4350 regression is solved */
-/* Force modal close button to be above modal content */
-:deep(.modal-container__close) {
-	z-index: 1;
+/* FIXME: remove after https://github.com/nextcloud-libraries/nextcloud-vue/pull/4959 is released */
+/* Styles to be applied outside of Talk (Deck plugin, e.t.c) */
+:deep(.modal-wrapper *) {
+	box-sizing: border-box;
 }
 
-.talk-modal {
-	height: 80vh;
+:deep(.modal-wrapper .modal-container) {
+	height: 100%;
+	max-height: 700px !important;
 }
```

Comment:
I highly suggest to use vh 
![image](https://github.com/nextcloud/spreed/assets/84044328/de57dfe9-8d76-4ac2-98ba-65f23636bd01)

It is too high for 1920 x 1080 display :/ and also the close button is unclickable because it is overlapping with the main header.

Reply:
Restored back original state. max-height should be as per default, and don't overlap with header:
```css
:deep(.modal-wrapper .modal-container) {
	height: 700px;
}
```

Reply:
Restored back original state. max-height should be as per default, and don't overlap with header:
```css
:deep(.modal-wrapper .modal-container) {
	height: 700px;
}
```